tests/unit/config/test_config_override_invariants.py

```
@@ -0,0 +1,507 @@
+    if isinstance(annotation, type) and issubclass(annotation, enum.Enum):
+        return list(annotation)
+    if annotation is bool:
+        return [True, False]
+    if annotation is int:
+        return [2, 3]
+    if annotation is float:
+        return [2.0, 3.0]
+    if annotation is str:
+        return ["aiperf-probe-a", "aiperf-probe-b"]
+    if annotation is Path:
+        return [Path("/tmp/aiperf-probe-a"), Path("/tmp/aiperf-probe-b")]
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/aiperf-probe-a"

(S108)